src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -258,6 +261,367 @@ public static async Task LargeJsonFile(int bufferSize)
                 Assert.Equal(SessionResponseCount, deserializedList.Count);
             }
         }
+
+        [Theory]
+//      [InlineData(1, true, true)]   ActiveIssue https://github.com/dotnet/corefx/issues/41598
```

Comment:
This issue has been fixed by https://github.com/dotnet/corefx/pull/42420. Can we re-enable these now?

Reply:
sure